plugins/ZProjector/src/main/java/org/micromanager/zprojector/CoordsComparator.java

```diff
@@ -14,12 +14,12 @@
 
    
    public CoordsComparator() {
-      cAxesNumeric.put(Coords.C, 1);
-      cAxesNumeric.put(Coords.CHANNEL, 1);
+      cAxesNumeric.put(Coords.T, 1);
+      cAxesNumeric.put(Coords.TIME_POINT, 1);
+      cAxesNumeric.put(Coords.C,3);
+      cAxesNumeric.put(Coords.CHANNEL, 3);
       cAxesNumeric.put(Coords.Z, 2);
       cAxesNumeric.put(Coords.Z_SLICE, 2);
-      cAxesNumeric.put(Coords.T, 3);
-      cAxesNumeric.put(Coords.TIME_POINT, 3);
```

Comment:
Perhaps this change was accidentally included?

Reply:
Regards to this the code is aligned to the one in origin/master. I think it is safe to keep.